### PR TITLE
Migrate application to nodejs

### PR DESCRIPTION
**Pull request by TooAngel**
Due to more experience in nodejs now and also a lot of issues in python. The application is migrated to nodejs.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -16,8 +16,6 @@ jobs:
       - run: npm install
```
Comment (mschultheiss83) [inline .circleci/config.yml:16]: - run: sudo pip install -r requirements.txt

**Review by mschultheiss83 — COMMENTED** [review]
no `  - run: sudo pip install -r requirements.txt` needed?

File: .circleci/config.yml (hunk 1 above)
Comment (TooAngel) [reply .circleci/config.yml:15]: Thx

**Comment by TooAngel** [comment]
This pull request was merged by [worlddriven](https://www.worlddriven.org).

**Pull request merged**